src/thx/schema/SimpleSchema.hx

```diff
@@ -66,6 +64,12 @@ class SimpleSchema {
       constAlt("none", None, function(a: Option<A>, b: Option<A>) return a == b)
     ]);
 
+  public static function makeNullable<E, A>(s: Schema<E, A>): Schema<E, Null<A>>
+    return oneOf([
+      alt("value", s, function(a: A) return a, function(a: Null<A>) return Options.ofValue(a)),
+      constAlt("novalue", null, function(a: A, b: A) return a == b)
```

Comment:
True, structural equality can be achieved with `thx.Dynamcs.equals`. Will do that. It is also true that we are comparing to a `null` value so strict equality should always work?